File: JAVA/2938. Separate Black and White Balls.java
class Solution {
    public long minimumSteps(String s) {
        long c=0;
        int i,w=0;
        for(i=0;i<s.length();i++){
            if(s.charAt(i)=='0'){
                c+=i-w;
                w++;
            }
        }
        return c;
    }
}

File: C++/2938. Separate Black and White Balls.cpp
class Solution {
public:
    long long minimumSteps(string s) {
    long long c=0;
    int i,w=0;
    for(i=0;i<s.size();i++){
        if(s[i]=='0'){
            c+=i-w;
            w++;
        }
    }
        return c;
    }
};